install.sh

```diff
@@ -434,20 +434,18 @@ main ()
 {
     jm_source="$PWD"
     jm_root="${jm_source}/jmvenv"
-    jm_deps="${jm_source}/deps"
```

Comment:
What `jm_deps` used at all outside the install script? Shellcheck didn't detect it being used.